plugins/tagoio-integration/src/back/RealtimeConnection.ts

```diff
@@ -16,14 +16,15 @@ const pingInterval = 240000;
 
 /**
  */
-function startRealtimeCommunication(token: string) {
+async function startRealtimeCommunication(token: string) {
   if (!token || events) {
     return;
   }
 
   const channel = "commands";
   const params = new URLSearchParams({ token, channel });
-  const url = `${getConnectionURI.default.default("usa-1").sse}?${params.toString()}`;
+  const region = (await pluginStorage.get("region")) || "us-e1";
+  const url = `${getConnectionURI.default.default(region).sse}?${params.toString()}`;
```

Comment:
Try changing this import to fix the `.default.default` below
Try changing import on top of file to this:

```ts
import getConnectionURI from "@tago-io/sdk/lib/regions.ts";
```

And remove `.default.default` from this line